iree/hal/vmla/op_kernels_test.cc

```diff
@@ -536,6 +536,49 @@ TEST(Conv2d, DepthwiseConv) {
   }
 }
 
+TEST(Transpose, 2Dimen) {
+  Shape src_shape = {2, 3};
+  Shape dst_shape = {3, 2};
+  std::vector<int32_t> perm = {1, 0};
+  // clang-format off
+  std::vector<uint16_t> src_buffer = { 1,  2,   3,
+                                      4,  5,   6};
+  std::vector<uint16_t> expected_dst = { 1,  4,
+                                         2,  5,   
+                                         3,  6};
```

Comment:
nit: if you disable clang-format for some lines, at least make them pretty ;D
```suggestion
  std::vector<uint16_t> src_buffer = {1, 2, 3,
                                      4, 5, 6};
  std::vector<uint16_t> expected_dst = {1, 4,
                                        2, 5,
                                        3, 6};
```

You could also work with clang-format by inserting comments to block its standard line wrapping:

```c++
  // (no clang-format off)
  std::vector<uint16_t> src_buffer = {1, 2, 3,  //
                                      4, 5, 6};
  std::vector<uint16_t> expected_dst = {1, 4,  //
                                        2, 5,  //
                                        3, 6};
```

Reply:
I've updated the indexing and also surrounded the values in block quotes to disable clang format.